### PR TITLE
Bugfix Bonanza 2

### LINKED ISSUE #1101: Rework plasteel alloy
By current material code from bay, 1/3 of the ship is made of fucking platinum, because plasteel is 1/2 steel and 1/2 platinum. Which is bullshit, we need to change it.

### LINKED ISSUE #2981: You can stick a gun into your mouth, even if something would be in the way.
_**Markus Corrales sticks their gun in their mouth, ready to pull the trigger...**
Markus Corrales pulls the trigger._

This is **Markus Corrales**!
They are wearing  a mining voidsuit helmet on their head.

### LINKED ISSUE #3171: Ghosts voting power
BUG: Ghosts get voting power 0.3, drones 1

### LINKED ISSUE #3380: [BUG] Cleaning ends prematurely
BUG: Moving to end the cleaning process with soap instantly cleans the floor without having to finish the progress bar

### LINKED ISSUE #3410: [BUG] You can implant cruciform into every living creature
Maybe we should stick only to humans, no?

### LINKED ISSUE #3416: [BUG] Airlocks not repairable
BUG: You can't repair doors. You can fit metal, but welder won't weld.

### PR DESCRIPTION
**Pull request by mistyLuminescence**
- Drones and mice now have low voting weight, similar to ghosts. You can't spawn as a drone to get more vote power.
- Plasteel is now made from (pla)sma and steel, not (pla)tinum and steel. Similarly, plasmaglass (borosilicate) is made from plasma, not platinum.
- When committing suicide with a gun, you now point it at your head, not in your mouth.
- Airlocks can now be repaired as intended- use metal on them, then a welding tool (or crowbar to cancel).
- Cleaning things with soap now respects do_after correctly.
- You can't use a cruciform on a non-humanoid target. Stop trying.
- Fixes a ton of typos in the tips list.

Fixes #1101 .
Fixes #2981 .
Fixes #3171 .
Fixes #3380 .
Fixes #3410 .
Fixes #3416 .

### CODE REVIEW

**Comment by Kurgis** [comment]
> 
>     * When committing suicide with a gun, you now point it at your head, not in your mouth.
> 

~~Feels a bit retroactive to me.~~ After viewing the code, this seems a bit odd, the code still handles it as being required to aim for the mouth, the only difference made is that warning text has their mentions of "mouth" replaced with "head". I'd recommend placing in checks whether mouth is covered instead, and disagree that the changes fix #2981 though it will make it harder to notice.

File: code/modules/tips_and_tricks/jobs.dm [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
 
 /tipsAndTricks/jobs/guild_resale_profit
     jobs_list = list(/datum/job/merchant, /datum/job/cargo_tech)
-    tipText = "As a guild merchant, you should be buying up valueable things from scavengers and reselling them for a profit. Buy anything of value that's brought to you"
+    tipText = "As a guild merchant, you should be buying up valuable things from scavengers and reselling them for a profit. Buy anything of value that's brought to you."
 
 /tipsAndTricks/jobs/captain_leader
     jobs_list = list(/datum/job/captain)
-    tipText = "A good leader's orders are always obeyed because a good leader never gives an order that won't be obeyed."
+    tipText = "A good leader's orders are always obeyed- because a good leader never gives an order that won't be obeyed."
```
Comment (Kurgis) [inline code/modules/tips_and_tricks/jobs.dm]: Is the dash meant to be a comma, or are you adding emphasis to the second half of the sentence?

Comment (mistyLuminescence) [reply code/modules/tips_and_tricks/jobs.dm]: Yeah, the latter.

**Comment by mistyLuminescence** [comment]
Having the text label it as 'head' while aiming at the mouth is consistent with aiming guns at other people- and it's weird to point a gun into someone else's mouth when their head is right there. Needing to select the mouth is just a gameplay abstraction to stop you from accidentally executing people when you're trying to double-tap them in combat.

**Pull request merged**